packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -1901,9 +1904,19 @@ function commitRootImpl(root, renderPriorityLevel) {
     // The first phase a "before mutation" phase. We use this phase to read the
     // state of the host tree right before we mutate it. This is where
     // getSnapshotBeforeUpdate is called.
-    prepareForCommit(root.containerInfo);
+    const activeInstanceFiber = prepareForCommit(root.containerInfo) || null;
```

Comment:
Hmm. So far we’ve almost avoided having Fibers be part of host config signatures. Except for events. Is it unavoidable here?

Reply:
If we pass back the host element then we need to ask the host for its instance again, so it's pretty unavoidable here.